src/modlunky2/mem/entities.py

```diff
@@ -202,17 +202,17 @@ def name(self):
 @dataclass(frozen=True)
 class Entity:
     type: Optional[EntityDBEntry] = struct_field(0x08, pointer(dc_struct))
+    overlay: EntityWrapper = struct_field(0x10, sc_void_p)
     items: Optional[Tuple[int, ...]] = struct_field(0x18, vector(sc_uint32))
     layer: int = struct_field(0x98, sc_uint8)
-    overlay: EntityWrapper = struct_field(0x10, sc_void_p)
 
 
 @dataclass(frozen=True)
 class Movable(Entity):
     state: CharState = struct_field(0x10C, sc_uint8)
+    holding_uid: int = struct_field(0x108, sc_int32)
```

Comment:
heh, this should be one higher :D

Reply:
all the more reason to add a test... the better to avoid mild embarrassment